tools/travis.ps1

```diff
@@ -149,7 +149,10 @@ else
         $ProgressPreference = $originalProgressPreference    
     }
 
-    $pesterParam = @{ 'binDir' = $output }
+    $pesterParam = @{ 
+        'binDir'   = $output 
+        'PassThru' = $true
+    }
```

Comment:
Why does this issue only happen to Travis CI? I assume it doesn't happen in AppVeyor as AppVeyor.psm1 is not chnaged to use the new parameter `-PassThru`.